Fixed the email link in index and readme

GeorgeKerscher opened the pull request:
I found the correct markdown to use to put in a email address. Now clicking on the amil should launch your default email client. I changed in readme and index.md.

I thought  I would include Mary  as reviewer in the PR just for experience. However, I did not see her name in the list. may not have rights. I can look into that.

brichwin commented:
Hi, George-
  In the README.md file, there is an extra great than sign immediately after the opening less than sign around the email address. It is causing all of the less than and greater than signs to render. In the index.md file, it is correct but there is a second version of the email address 5 lines down. 
   It appears that we might be able to simply place an email address in inline with the body text of a paragraph an GH recognizes it automatically and places the correct link around it.

GeorgeKerscher commented:
I removed the extra markup in readme. I removed the extra email in the index.md. 

I did not try putting in a plain email expecting GitHub to put in the markup.

Should we remove the index.md, because it is just an extra file to maintain and it should link to the index in the docs file like the index does now?

brichwin reviewed (approved):
Looks good!
